### PR TITLE
2024 11 20 main package improvements

### PR DESCRIPTION
**Pull request by thomas-roos**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/build-packages-multi-platform-container-qemu/readme.template.txt [hunk 1]
```diff
@@ -1,58 +1,64 @@
...
 sudo ./install-greengrass-lite.sh -u
 
 ###############################################################################
-#
+# UPGRADE
+###############################################################################
+
+When a new version of greengrass lite is available and you want to keep your
+configuration and components. Install just the deb package from the zip, without
+using the (inital-) installation script.
```
Comment (amazonKamath) [inline .github/workflows/build-packages-multi-platform-container-qemu/readme.template.txt:56]: *initial?

**Review by amazonKamath — APPROVED** [review]

File: .github/workflows/build-packages-multi-platform-container-qemu/readme.template.txt (hunk 1 above)
Comment (thomas-roos) [reply .github/workflows/build-packages-multi-platform-container-qemu/readme.template.txt:56]: The script is just for the initial installation. Not for an upgrade.

Comment (amazonKamath) [reply .github/workflows/build-packages-multi-platform-container-qemu/readme.template.txt:56]: Sorry, I was not clear. I was talking about typo.

Comment (thomas-roos) [reply .github/workflows/build-packages-multi-platform-container-qemu/readme.template.txt:56]: Ah - got it now.
My brain did auto correct it :D

**Pull request merged**